### PR TITLE
Add a CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by jerryaldrichiii**
This adds a contributor code of conduct similar to:

https://github.com/habitat-sh/habitat/blob/master/CODE_OF_CONDUCT.md

@jquick when you get back from PTO, let me know if you want to be on the list of Advocates. I didn't want to add you without asking. :smile:

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+## Enforcement/Getting Help
+
+Instances of abusive, harassing, or otherwise unacceptable behavior should be reported by contacting any of the Community Advocates directly. Each person's contact information and role is listed below. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+Community Organizers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+
+## Roles
+
+The following are the various roles of our <b>Community Organizers</b> and the person(s) assigned to each role:
+  * The <b>Decider</b> has final say on community guidelines and final authority on correct actions and appeals. The top-level decider is [Adam Jacob](mailto:adam@habitat.sh), [@adamhjk](https://twitter.com/adamhjk)
```
Comment (TrevorBramble) [inline CODE_OF_CONDUCT.md]: @jerryaldrichiii Should this be @chris-rock?

**Comment by miah** [comment]
Very +1 on this, we should also add a link to the Code of Conduct from:

https://github.com/inspec/inspec/blob/master/ISSUE_TEMPLATE.md
https://github.com/inspec/inspec/blob/master/CONTRIBUTING.md
https://github.com/inspec/inspec/blob/master/README.md

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (jerryaldrichiii) [reply CODE_OF_CONDUCT.md]: Hmmm...you make a good point. I'll ask him...by linking here :smile:

**Comment by jerryaldrichiii** [comment]
Great idea @miah! I added it in the latest commit.

**Review by miah — APPROVED** [review]

**Review by clintoncwolfe — APPROVED** [review]
Approve, and I commit to being a Community Advocate.

**Comment by jerryaldrichiii** [comment]
I added @chris-rock and @arlimus as Deciders per their request.

**Comment by jquick** [comment]
This looks good. You can throw me on the Community Advocates also! 

Thanks for this @jerryaldrichiii

**Pull request merged**